clients/cobol-lsp-vscode-extension/src/provider/TarCopybookFileSystemProvider.ts

```diff
@@ -0,0 +1,176 @@
+import { Minimatch } from "minimatch";
+import * as vscode from "vscode";
+import { Memoize } from "../services/util/Memoize";
+import { TarUtil } from "../services/util/TarUtil";
+
+export class TarCopybookFileSystemProvider
+  implements vscode.FileSystemProvider
+{
+  public static SCHEME = "tar";
```

Comment:
I think that this scheme might lead to conflicts with other extensions = perhaps `cobol-ls-tar`?
```suggestion
  public static readonly SCHEME = "cobol-ls-tar";
```

Reply:
should resolved at : c1853f05dcc2b78616234efef59b5d531ef017b1